### ISSUE TITLE
Make the submucosal glands of the respiratory tract part of the respiratory epithelium (they are continuous with it).

### ISSUE THREAD

**Issue by dosumis** [obophenotype/uberon#3282]
Rationale: the submucosal glands of the respiratory tract are continuous with (part of) the respiratory tract epithelium, but this is not reflected in Uberon - this results in disjointed hierarchy that splits of submucosal gland cell types from similar cells in the rest of the respiratory epithelium. Note - It is OK to have some subdivision of the respiratory epithelium that divides these, but we need a general term under which the general cell type exist.

**Comment by aleixpuigb** [comment]
I agree that the submucosal gland eputhelium is part of the respiratory epithelium. However, the submucosal gland is more than the epithelium. I propose create `epithelium of submucosal gland` and that should solve the problem.

**Comment by dosumis** [comment]
Superceded by:

#3491 - which asks for the opposite
